tox.ini

```diff
@@ -130,4 +130,4 @@ commands =
 deps =
     # If you need to test on a certain branch, add @<branch-name> after .git
     git+https://github.com/amazon-braket/amazon-braket-schemas-python.git
-    git+https://github.com/amazon-braket/amazon-braket-default-simulator-python.git
+    git+https://github.com/amazon-braket/amazon-braket-default-simulator-python.git@multi2
```

Comment:
reminder to update after https://github.com/amazon-braket/amazon-braket-default-simulator-python/pull/264 is merged and released